generators/generator-bot-core-assistant/generators/app/templates/dialogs/BotTourDialog/BotTourDialog.dialog

```diff
@@ -34,79 +34,62 @@
       "intent": "DetailsCard",
       "actions": [
         {
-          "$kind": "Microsoft.ChoiceInput",
+          "$kind": "Microsoft.SendActivity",
           "$designer": {
-            "id": "8kK1VI"
+            "id": "ZMzFeP"
           },
-          "defaultLocale": "en-us",
-          "disabled": false,
-          "maxTurnCount": 3,
-          "alwaysPrompt": false,
-          "allowInterruptions": true,
-          "unrecognizedPrompt": "",
-          "invalidPrompt": "",
-          "prompt": "${ChoiceInput_Prompt_BotTourDetails()}",
-          "choiceOptions": {
-            "includeNumbers": true,
-            "inlineOrMore": ", or ",
-            "inlineOr": " or ",
-            "inlineSeparator": ", "
+          "activity": "${SendActivity_BotTourDetails()}"
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnIntent",
+      "$designer": {
+        "id": "vdn2Dw",
+        "name": "ActionA"
+      },
+      "intent": "ActionA",
+      "actions": [
+        {
+          "$kind": "Microsoft.SendActivity",
+          "$designer": {
+            "id": "oeL3EC"
           },
-          "choices": [
-            "A",
-            "B",
-            "C"
-          ],
-          "style": "none",
-          "recognizerOptions": {
-            "recognizeOrdinals": true
+          "activity": "${SendActivity_SuggestedActionAContent()}"
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnIntent",
+      "$designer": {
+        "id": "ruLbcx",
+        "name": "ActionB"
+      },
+      "intent": "ActionB",
+      "actions": [
+        {
+          "$kind": "Microsoft.SendActivity",
+          "$designer": {
+            "id": "PSsrvD"
           },
-          "property": "user.botTourDetailsChoice"
-        },
+          "activity": "${SendActivity_SuggestedActionBContent()}"
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnIntent",
+      "$designer": {
+        "id": "Fris5p",
+        "name": "ActionC"
+      },
+      "intent": "ActionC",
```

Comment:
Should there be dialog where the intent gets executed?

Reply:
I'm not sure what you mean... this has a $kind of Microsoft.OnIntent, with an intent ("DetailsCard") and an array of actions on how to respond (in this case it just sends an activity). There doesn't need to be an additional reference to a dialog, it's in a dialog itself.

Reply:
Ah got it, sorry I misread the json. :)

This is what happens when I am trying to visualize the json in the composer. :(